src/atoms/Button/index.js

```diff
@@ -49,13 +47,18 @@ Button.propTypes = {
   type: PropTypes.string,
 
   /**
-   * Possible button variants are: `info`, `disabled`, `toggle`, `action`, `actionDisabled`, `lowlight`, `outline`, or `button` (default)
+   * Possible button variants are: `info`, `disabled`, `outline` (deprecated)
```

Comment:
what exactly is deprecated here? You can add a deprecation warning with a PropTypes function for specific keywords if necessary

Reply:
The "outline" is deprecated.